homeassistant/components/tedee/config_flow.py

```diff
@@ -45,9 +58,26 @@ async def async_step_user(
             step_id="user",
             data_schema=vol.Schema(
                 {
-                    vol.Required(CONF_HOST): str,
-                    vol.Required(CONF_LOCAL_ACCESS_TOKEN): str,
+                    vol.Required(
+                        CONF_HOST,
+                        default=self.reauth_entry.data[CONF_HOST]
+                        if self.reauth_entry
+                        else "",
+                    ): str,
+                    vol.Required(
+                        CONF_LOCAL_ACCESS_TOKEN,
+                        default=self.reauth_entry.data[CONF_LOCAL_ACCESS_TOKEN]
```

Comment:
Does prefilling the token provide any value? I mean it changed so there is no real reason to know what used to be there

Reply:
I had it in the past that sometimes integration report a reauth error but it's solved on resubmit

Reply:
And with tedee that can happen if you turn a switch to enable unencrypted keys in the Bridge settings - the key stays the same tough, so you can just reauth later